README.md

```diff
@@ -68,12 +69,12 @@ NOTE: This example is superseded by the [NioCommand](https://github.com/nio-bloc
 
 HTTPRequestsPostSignal
 ======================
-Similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
+The HTTPRequestsPostSignal block is similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
 
 Properties
 ----------
 - **basic_auth_creds**: When making a request that needs Basic Authentication, enter the username and password.
```

Comment:
Optionally enter the username and password when making a request that requires basic authentication.